src/fluree/db/query/exec/eval.cljc

```diff
@@ -749,6 +751,25 @@
                             (where/->typed-val nil)
                             args))
 
+           ;; iri function for SCI
+           ;; This is the base two-parameter function
+           iri-fn-base (fn [input ctx]
+                         (log/debug "iri-fn-base called with input:" input "ctx:" (some? ctx))
+                         (let [value (if (map? input)
+                                       (:value input)
+                                       input)
+                               ;; Special handling for @type
+                               expanded (if (= "@type" value)
+                                          const/iri-rdf-type
```

Comment:
Like in the other case, I think we switched from using `iri-rdf-type` to `iri-type` internally (see `fql.parse/parse-pred-cmp`).